typesafe/src/main/scala/knobs/Typesafe.scala

```diff
@@ -22,61 +22,65 @@ import scalaz.concurrent.Task
 
 object Typesafe {
   def config: Task[Config] = Task {
-    def convertList(list: ConfigList): CfgList = {
-      def unwrap[T](c: ConfigValue)(implicit ev: ClassTag[T]): T =
-        c.unwrapped match {
-          case t: T => t
-          case _ =>
-            sys.error(s"Can't convert $c to underlying type ${ev.runtimeClass.getName}")
-        }
+    val cfg = ConfigFactory.load
+    Config(convertTypesafeConfig(cfg))
+  }
 
-      val items: List[CfgValue] =
-        list.toList.flatMap { v =>
-          v.valueType match {
-            case ConfigValueType.NULL => None
-            case ConfigValueType.BOOLEAN =>
-              Some(CfgBool(unwrap[Boolean](v)))
-            case ConfigValueType.LIST =>
-              Some(convertList(unwrap[ConfigList](v)))
-            case ConfigValueType.NUMBER =>
-              Some(CfgNumber(unwrap[Number](v).doubleValue))
-            case ConfigValueType.STRING =>
-              Some(CfgText(unwrap[String](v)))
-            case x =>
-              sys.error(s"Can't convert $v to a CfgValue")
-          }
+  def config(cfg: TC): Task[Config] = Task {
```

Comment:
I think the only reason this is in task is because the typical usage is at the edge of the world and it was convenient to sequence them in `Task`, but otherwise, i dont think there's any specific reason its in `Task`